tools/dependency/cve_scan.py

```diff
@@ -7,6 +7,7 @@
 import datetime as dt
 import gzip
 import json
+import os
```

Comment:
Seems unrelated and that it should have been broken before if not there.

Reply:
removed